internal/services/containers/kubernetes_cluster_node_pool_resource.go

```diff
@@ -164,6 +164,13 @@ func resourceKubernetesClusterNodePoolSchema() map[string]*pluginsdk.Schema {
 			}, false),
 		},
 
+		"gpu_driver": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringInSlice(agentpools.PossibleValuesForGPUDriver(), false),
```

Comment:
If this is omitted, is a default value returned from the API? (e.g. does it return `GPUDriverNone` / `"None"` by default) If so, this could cause an unexpected forced recreation of existing resources.